vault/dendron.topic.refactoring.commands.move-note.md

```diff
@@ -20,5 +20,13 @@ none
 
 **{{fm.title}}** is a command that lets you move a note between vaults when in a [[Multi-vault|dendron://dendron.dendron-site/dendron.topic.multi-vault]] workspace, and will also update all references in other notes of that note.
 
-## Moving a note using the file explorer
+1. Call **{{fm.title}}** from the command bar
+2. Type the note title
+   1. (Optional) If you want to move a note and all their children, type the parent's note and click on the square icon on the right of the panel (see image below)
+3. Select the vault you want to move the note(s) to
+
+![**{{fm.title}}**: Selecting several notes at once](assets/images/move-note.png)
```

Comment:
Thanks for the PR. Feel free to add this link: https://org-dendron-public-assets.s3.amazonaws.com/images/move-multiple-notes.png

Reply:
Thanks for the correct link! Now added in [d1501b7](https://github.com/dendronhq/dendron-site/pull/623/commits/d1501b7b7800406a8342a1b3f5b01ef12257fcbe)